mne/beamformer/tests/test_lcmv.py

```diff
@@ -746,52 +746,58 @@ def test_lcmv_reg_proj(proj, weight_norm):
         assert_allclose(stc_cov.data.std(), 0.12, rtol=0.1)
 
 
-@pytest.mark.parametrize('reg, weight_norm, use_cov, lower, upper', [
-    (0.05, 'unit-noise-gain', True, 96, 98),
+@pytest.mark.parametrize('reg, weight_norm, use_cov, depth, lower, upper', [
     # the 0 reg is not so stable, can produce a wide range of scores
```

Comment:
Mh, did not catch this in the last round - why is that? Is the data here rank-deficient?

Reply:
Not sure, feel free to add to the beamforming agenda. But the range was like 20% or something across our CIs

Reply:
Is the data randomly generated or why does it differ? Or does it depend on environment? If random data is not the case, the results should as least be stably wrong ...

Reply:
> why does it differ

Numerical differences across OS / LAPACK implementations, i.e. probably tiny values around machine precision eventually getting blown up.

Reply:
Mh, I wonder whether we are running into problems similar to #4628 ... I think would be valuable to check out whether this is the case or if data are rank-deficient etc. I'll put it into our agenda issue!